R/check_lintr.R

```diff
@@ -1,5 +1,24 @@
 #' Check the packages for linters
 #'
+#' This functions does [static code analysis](
+#' https://en.wikipedia.org/wiki/Static_program_analysis).
+#' It relies on [lintr::lint_package()].
+#' We recommend that you activate all code diagnostics in RStudio.
```

Comment:
Explaining why something is recommended, seems helpful here and motivating to do the effort.  (Why doing something if you even don't know why it is useful?)  I'm not sure my suggestion is the best way to explain it, so please reword.

```suggestion
#' We recommend that you activate all code diagnostics in RStudio to help meeting the requirements.
```